sklearn/tree/_splitter.pyx

```diff
@@ -240,13 +250,6 @@ cdef class BaseDenseSplitter(Splitter):
 
 cdef class BestSplitter(BaseDenseSplitter):
     """Splitter for finding the best split."""
-    def __reduce__(self):
```

Comment:
note to other reviewers: we can remove these since Cython adds `__reduce__` if there's no `__cinit__` present: https://cython.readthedocs.io/en/latest/src/userguide/extension_types.html#controlling-pickling